Gemfile

```diff
@@ -11,6 +11,6 @@ end
 group :debug do
   gem "pry"
   gem "pry-byebug"
-  gem "pry-stack_explorer"
+  gem "pry-stack_explorer", "~> 0.4.13" # pin until we support ruby 2.5
```

Comment:
Do we need to pin pry-stack_explorer?

Reply:
Yes @jeremiahsnapp latest pry-stack-explorer doesn't support ruby 2.5